### DISCUSSION TITLE
Verschiedene Fragen anhand von Fallbeispielen zur Ereignis-Annotation in Grundbuch-Dokumenten

### DISCUSSION THREAD

**Discussion by raykyn** [DHBern/BeNASch#9] [Edge cases]
Da wir im Moment dabei sind, die Leitfäden zur Ereignis-Annotation zu schreiben, kommen immer wieder Fragen zu komplexen Fällen auf. Damit sich diese nicht in irgendwelchen Slack- oder Discordchats verlieren, verschieben wir die Diskussion dieser Fälle auf diesen Thread. 

Auch wenn hier wohl hauptsächlich Mitglieder des Projekts Ökonomien des Raums posten werden, sind auch alle anderen Interessierten eingeladen, sich an der Diskussion zu beteiligen. (Wobei wir dazu wohl noch die WIP-Ereignis-Annotation veröffentlichen müssen).

**Comment by behitz** [comment]
Am Schluss eines Rentenkaufs steht folgende Klausel: "iren 4 unschädlich und gelöbt den kouff ze werende und state ze hande ." Es ist nun nicht ganz klar, ob dieses Versprechen ein eigentliches Ereignis ist und annotiert werden soll. Oder eher, dass dies einfach Teil der Formel des Rentenkaufs ist und allenfalls eine Rolle im Kauf hat – wenn überhaupt? Aktuell besteht die Tendenz zu Letzterem und das Einzige, was annotiert ist, ist die Erwähnung des Kaufereignisses mit Korreferenz auf den Rentenkauf.
